### PR TITLE
Fix pre-commit checks in CI

### PR DESCRIPTION
**Pull request by ryanjulian**
This commit message intentionally violates the style guideeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -7,48 +7,84 @@ repos:
...
     # -   id: sort-simple-yaml            # Sorts simple YAML files which consist only of top-level keys, preserving comments and blocks.
+    #     stages: [commit, push]
     -   id: trailing-whitespace         # Trims trailing whitespace.
+        stages: [commit, push]
+
 -   repo: https://github.com/pre-commit/mirrors-yapf
     rev: v0.22.0
     hooks:
     -   id: yapf
-        args: ['-dpr']
```
Comment (jonashen) [inline .pre-commit-config.yaml]: Are you sure you can remove args? I had to keep them in order for both yapf and pylint to run correctly with pre-commit

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -7,48 +7,84 @@ repos:
...
+        stages: [commit, push]
+
 -   repo: https://github.com/pre-commit/mirrors-pylint
     rev: v1.9.1
+    stages: [commit, push]
     hooks:
     -   id: pylint
-        args: ['--rcfile=setup.cfg']
```
Comment (jonashen) [inline .pre-commit-config.yaml]: I think you need to keep args

Comment (jonashen) [reply .pre-commit-config.yaml]: Or else pre-commit will say `No files to check(skipped)`

Comment (ryanjulian) [reply .pre-commit-config.yaml]: Oops not sure how these got removed

**Comment by jonashen** [comment]
Is `garage/scripts/check_commit_message` extraneous now?

**Comment by ryanjulian** [comment]
No, it's still used by pre-commit

**Pull request merged**